### ISSUE TITLE
Prometheus connector always requires `labels` column

### ISSUE THREAD

**Issue by ebyhr** [trinodb/trino#12301]
```sql
trino> SELECT * FROM prometheus.default.up;
                         labels                         |           timestamp            | value
--------------------------------------------------------+--------------------------------+-------
 {instance=localhost:9090, job=prometheus, __name__=up} | 2022-05-10 09:43:54.478 +09:00 |   1.0

trino> SELECT labels FROM prometheus.default.up;
                         labels
--------------------------------------------------------
 {instance=localhost:9090, job=prometheus, __name__=up}

trino> SELECT labels, value FROM prometheus.default.up;
                         labels                         | value
--------------------------------------------------------+-------
 {instance=localhost:9090, job=prometheus, __name__=up} |   1.0

trino> SELECT "timestamp" FROM prometheus.default.up;
Query 20220510_004758_00006_a3dez failed: labels is null

trino> SELECT value FROM prometheus.default.up;
Query 20220510_005139_00017_a3dez failed: labels is null
```

```
Query 20220510_004817_00007_a3dez failed: labels is null
java.lang.NullPointerException: labels is null
	at java.base/java.util.Objects.requireNonNull(Objects.java:246)
	at io.trino.plugin.prometheus.PrometheusStandardizedRow.<init>(PrometheusStandardizedRow.java:30)
	at io.trino.plugin.prometheus.PrometheusRecordCursor.lambda$prometheusResultsInStandardizedForm$0(PrometheusRecordCursor.java:191)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at io.trino.plugin.prometheus.PrometheusRecordCursor.lambda$prometheusResultsInStandardizedForm$1(PrometheusRecordCursor.java:192)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at io.trino.plugin.prometheus.PrometheusRecordCursor.prometheusResultsInStandardizedForm(PrometheusRecordCursor.java:194)
	at io.trino.plugin.prometheus.PrometheusRecordCursor.<init>(PrometheusRecordCursor.java:78)
	at io.trino.plugin.prometheus.PrometheusRecordSet.cursor(PrometheusRecordSet.java:58)
	at io.trino.spi.connector.RecordPageSource.<init>(RecordPageSource.java:37)
	at io.trino.split.RecordPageSourceProvider.createPageSource(RecordPageSourceProvider.java:50)
	at io.trino.split.PageSourceManager.createPageSource(PageSourceManager.java:68)
	at io.trino.operator.TableScanOperator.getOutput(TableScanOperator.java:308)
	at io.trino.operator.Driver.processInternal(Driver.java:388)
	at io.trino.operator.Driver.lambda$processFor$9(Driver.java:292)
	at io.trino.operator.Driver.tryWithLock(Driver.java:693)
	at io.trino.operator.Driver.processFor(Driver.java:285)
	at io.trino.execution.SqlTaskExecution$DriverSplitRunner.processFor(SqlTaskExecution.java:1092)
	at io.trino.execution.executor.PrioritizedSplitRunner.process(PrioritizedSplitRunner.java:163)
	at io.trino.execution.executor.TaskExecutor$TaskRunner.run(TaskExecutor.java:488)
	at io.trino.$gen.Trino_testversion____20220510_004343_2.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by Emor-nj** [comment]
I want to work on this issue, assign to me for first issue to fix. Thanks!

**Comment by ebyhr** [comment]
Resolved by #12510
